<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fb2Find</class>
 <widget class="QDialog" name="Fb2Find">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>128</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,1" columnstretch="0,0,1,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Find what:</string>
       </property>
       <property name="buddy">
        <cstring>editText</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editText"/>
     </item>
    </layout>
   </item>
   <item row="0" column="3" rowspan="4">
    <layout class="QVBoxLayout" name="verticalLayout2">
     <item>
      <widget class="QPushButton" name="btnFind">
       <property name="text">
        <string>Find &amp;next</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticallSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupDirect">
     <property name="title">
      <string>&amp;Direction</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout3">
      <item>
       <widget class="QRadioButton" name="radioUp">
        <property name="text">
         <string>&amp;Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDown">
        <property name="text">
         <string>&amp;Down</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkCase">
       <property name="text">
        <string>&amp;Case sensitive </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkHigh">
       <property name="text">
        <string>&amp;Highlights all</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editText</tabstop>
  <tabstop>checkCase</tabstop>
  <tabstop>checkHigh</tabstop>
  <tabstop>radioUp</tabstop>
  <tabstop>radioDown</tabstop>
  <tabstop>btnFind</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>Fb2Find</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
